config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics.yaml

```diff
@@ -41,3 +41,30 @@ periodics:
   annotations:
     testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
     testgrid-tab-name: periodic-conformance-v1alpha3
+- name: periodic-cluster-api-provider-azure-capi-e2e
+  path_alias: "sigs.k8s.io/cluster-api-provider-azure"
+  decorate: true
+  decoration_config:
+    timeout: 4h
+  interval: 12h
+  max_concurrency: 5
```

Comment:
This line also shouldn't be necessary since there is at most one job running at any given time.